teradatalabs/presto-cli/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM java:8-jre
+ARG PRESTO_VERSION
+
+ADD http://teradata-presto.s3.amazonaws.com/presto/$PRESTO_VERSION/presto-cli-$PRESTO_VERSION-executable.jar /usr/local/bin/presto
+
+RUN chmod +x /usr/local/bin/presto
+
+RUN apt-get update && \
```

Comment:
don't do the update - it makes the image output nondeterministic (dependent on the time you build it). Just rely on the ubuntu image being reasonably fresh